### ISSUE TITLE
Default variant for `Callout` does not meet WCAG color contrast requirements for certain colors

### ISSUE THREAD

**Issue by dbollinger** [radix-ui/themes#694]
## Context

Component: [Callout ](https://www.radix-ui.com/themes/docs/components/callout)

## Expected behavior

Default text styles should meet WCAG color contrast requirements

## Actual behavior

Color contrast fails for `color="red"` and `color="green"`

## Workarounds

Use `highContrast` option
